secondary-server-1/src/main/java/com/replicated_log/secondary_server_1/controller/rest/SecondaryRestController.java

```diff
@@ -0,0 +1,70 @@
+package com.replicated_log.secondary_server_1.controller.rest;
+
+import com.replicated_log.secondary_server_1.model.Ack;
+import com.replicated_log.secondary_server_1.model.AckStatusCode;
+import com.replicated_log.secondary_server_1.model.Item;
+import com.replicated_log.secondary_server_1.service.SecondaryClient;
+import com.replicated_log.secondary_server_1.service.ItemService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping(SecondaryRestController.SECONDARY_1_URL)
+@CrossOrigin("*")
+public class SecondaryRestController {
+
+    private final Logger LOG = LogManager.getLogger(SecondaryRestController.class);
+
+    public static final String SECONDARY_1_URL = "/secondary1";
+    public final String SECONDARY_SERVER_NAME = "Secondary1";
+
+    @Value("${secondary.server.baseurl}")
+    private String SECONDARY_SERVER_BASE_URL;
+
+    @Autowired
+    private ItemService itemService;
+
+    @Autowired
+    private SecondaryClient secondaryClient;
+
+    @GetMapping("/items")
+    public ResponseEntity<Set<Item>> findAllItems() {
+        Set<Item> items = itemService.getItems();
+        return ResponseEntity.ok(items);
+    }
+
+    @GetMapping("/items-demo")
+    public ResponseEntity<Set<Item>> findAllItemsForDemo() {
+        Set<Item> items = itemService.getItemsForDemo();
+        return ResponseEntity.ok(items);
+    }
+
+    @PostMapping(value = "/item")
+    public ResponseEntity<Item> addItem(@RequestBody Item item) {
+        LOG.info("C--> Received Item - " + item.getText());
+
+        LOG.info("C--> Set of " + SECONDARY_SERVER_NAME + ":" + itemService.getItems());
+        itemService.simulateProcessing(5);
```

Comment:
what is number 5 ?

Reply:
added constant 
private static final int PROCESSING_SECONDS = 5;